libs/core/coroutines/src/detail/context_posix.cpp

```diff
@@ -7,9 +7,10 @@
 #include <hpx/config.hpp>
 
 #if !defined(HPX_HAVE_GENERIC_CONTEXT_COROUTINES) &&                           \
-    !(defined(__linux) || defined(linux) || defined(__linux__)) &&             \
+    !defined(HPX_HAVE_FIBER_BASED_COROUTINES) &&                               \
```

Comment:
Don't the checks for the linux definitions have to stay?

Reply:
On Summit both macros, `__linux__` and `__powerpc__` are defined...

Reply:
@msimberg I have changed the conditions to match the ones used in context_impl.hpp. Things should work everywhere now.

Reply:
@msimberg yep, clang-newest is happy now as well. @diehlpk would you mind testing this one more time on summit?